bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/i18n/mybmw_de.properties

```diff
@@ -179,6 +179,7 @@ channel-type.mybmw.timer4-day-sun-channel.label = Zeitprofil 4 - Sonntag
 channel-type.mybmw.gps-channel.label = Koordinaten
 channel-type.mybmw.heading-channel.label = Ausrichtung
 channel-type.mybmw.address-channel.label = Adresse
+channel-type.mybmw.home-distance-channel.label = Entfernung von Zuhause
```

Comment:
Please remove the German translation from the pull request. It can be provided through Crowdin after the PR is merged. For reference: https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations

Reply:
I see. Remove whole file or just go back to the released version?

Reply:
Just revert the change, i.e. the version of the file before your PR.